apis/r/DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: TileDB SOMA
 Description: Interface for working with 'TileDB'-based Stack of Matrices,
     Annotated ('SOMA'): an open data model for representing annotated matrices,
     like those commonly used for single cell data analysis.
-Version: 1.4.3
+Version: 1.4.3.3
```

Comment:
Oopps. That was local, to differentiate from earlier build. I can (and likely should) take that out.

Reply:
And done.